### PR TITLE
Wrote a blog post

### PR DESCRIPTION
**Pull request by gpat-roo**
Wrote a blog post about dev giving estimates. It’s a good blog post. I
hope it makes the cut.

### CODE REVIEW

**Comment by kplattret** [comment]
Good post @gpat-roo!

File: _posts/2016-08-26-why-its-impossible.mdown [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+The builder in question has only worked on house 1, which is built on hard ground. House 3 was build on sand. It is perfectly fine, but as soon as you build a new room, it’ll start to sink. So if we really need this room, we’ll first need to shore up the foundations. The builder has no idea how long this will take as he’s not familiar with the foundations, and will need to understand them first.
+
+But let's say that the room required is simple, with well-defined specs, and the foundations are actually fine. The builder knows that similar rooms have taken 30 days. But the builder also knows that house 2 is more worth more, and there is a high likelihood of new rooms being needed for that in the next month as well. So even though it is 30 days of _actual_ work, it will likely take more than a month to deliver.
```
Comment (gbonfant) [inline _posts/2016-08-26-why-its-impossible.mdown]: s/is more worth more/is worth more

**Comment by gbonfant** [comment]
Good stuff although not sure how to feel about the builder analogy. It doesn't fit well with a developer not to mention you'd probably ask the architect those kind of questions.

File: _posts/2016-08-26-why-its-impossible.mdown [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title:  "Why it's impossible for devs to give dates"
+date:   2016-06-27 10:13:59
+exerpt: "Or 'Why your tiny change will take three weeks'"
+---
+
+_By Graham Paterson_
```
Comment (kapowaz) [inline _posts/2016-08-26-why-its-impossible.mdown]: I’m working on adding an `author` metadata field in the Front Matter, so I suggest moving this up to there.

**Comment by jazlalli** [comment]
One thing that struck me immediately was the frequency of occurrence of `he`, when referring to the builder. I wonder if this might only further __cement__ (😛) the notion of male's as builders/developers, which is obviously not true.

Given that the thought crossed my mind whilst reading, it's bound to for others too. We might want to be mindful of that.

**Comment by mezis** [comment]
> I can’t offer any great solution in instances where deadlines are required

Maybe - to conclude on a positive note - we'd actually want to mention solutions?
We (probably) all know there's exactly one solution to deadlines: accepting compromise on scope or on quality.

File: _posts/2016-08-26-why-its-impossible.mdown (hunk 2 above)
Comment (mezis) [inline _posts/2016-08-26-why-its-impossible.mdown]: On a more meta note, I for one welcome PMs to write article. But given the blog's called deliveroo._engineering_, might be worth mentioning the author's role!

**Comment by mezis** [comment]
@gbonfant:

> not sure how to feel about the builder analogy

+1 to that but I can live with it with one caveat:

While I appreciate not everyone is fussed by the distinction, the term used should be "engineer", not "dev(eveloper)" or "programmer". The latter two (tending to be equivalent) reduce us to the task (writing code); the former is about _process_ (figuring out how to build something, from an idea to delivery).

**Comment by kapowaz** [comment]
I asked @gpat-roo about this blog and he said he’d probably never get around to addressing the comments, so for now I’m going to close it but not delete the branch. If he gets time he can re-open the PR.

**Pull request closed without merging**